React native 0.64.0 support

Issue was brought up a couple months ago here: [1886](https://github.com/react-native-webview/react-native-webview/issues/1886)

Wanted to resurface this since this is causing build issues when I attempt to upgrade.

```
error "react-native-webview#react-native@>=0.60 <0.64" doesn't satisfy found match of "react-native@0.64.0"
```

Try to npm install react-native-webview@11.3.2

`npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: moovben@0.0.1
npm ERR! Found: react-native@0.64.0
npm ERR! node_modules/react-native
npm ERR!   react-native@"0.64.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native@">=0.60 <0.64" from react-native-webview@11.3.2
npm ERR! node_modules/react-native-webview
npm ERR!   react-native-webview@"11.3.2" from the root project`

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "type": "Component"
   },
   "peerDependencies": {
-    "react-native": ">=0.60 <=0.64"
+    "react-native": ">=0.60"
   },
   "dependencies": {
     "escape-string-regexp": "2.0.0",

